wp-content/themes/oceanwp-child/assets/css/template-styles.css

```diff
@@ -273,4 +276,10 @@ hr{
 .ini-supporters{
     color:#DC9C05;
     margin-bottom:0px
+}
+
+.ess-list{
```

Comment:
What is `ess`?

Reply:
easy social sharing, I've extended this css in initiatives share button. They're using ess prefix for their css classes.